plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -46,16 +46,9 @@ public function __invoke( OD_HTML_Tag_Walker $walker ): bool {
 		$common_lcp_element = $this->url_metrics_group_collection->get_common_lcp_element();
 		if ( ! is_null( $common_lcp_element ) && $xpath === $common_lcp_element['xpath'] ) {
 			if ( 'high' === $walker->get_attribute( 'fetchpriority' ) ) {
-				$walker->set_attribute( 'data-od-fetchpriority-already-added', true );
+				$walker->set_meta_attribute( 'fetchpriority-already-added', true );
 			} else {
 				$walker->set_attribute( 'fetchpriority', 'high' );
-				$walker->set_attribute( 'data-od-added-fetchpriority', true );
-			}
-
-			// Never include loading=lazy on the LCP image common across all breakpoints.
-			if ( 'lazy' === $walker->get_attribute( 'loading' ) ) {
-				$walker->set_attribute( 'data-od-removed-loading', $walker->get_attribute( 'loading' ) );
-				$walker->remove_attribute( 'loading' );
 			}
```

Comment:
This code is obsolete because the `loading` attribute is now removed for any element which has an non-zero `intersectionRatio` in any viewport group.